node/src/types/block.rs

```diff
@@ -261,8 +259,8 @@ impl Display for BlockPayload {
         write!(
             formatter,
             "block payload: deploys {}, transfers {}, accusations {:?}, random bit {}",
-            HexList(&self.deploy_hashes),
-            HexList(&self.transfer_hashes),
+            format!("[{}]", itertools::join(&self.deploy_hashes, ", ")),
+            format!("[{}]", itertools::join(&self.transfer_hashes, ", ")),
```

Comment:
If I'm reading this correctly, this will now print `deploy-hash(01234567..)` instead of just `01234567..`. I'd prefer keeping the logs as concise as possible.